code/modules/assembly/signaler.dm

```diff
@@ -145,8 +145,9 @@
 		manual_suicide(suicider)
 		return
 	pulse(TRUE)
-	audible_message("[icon2html(src, hearers(src))] *beep* *beep* *beep*", null, hearing_range)
-	playsound(get_turf(src), 'sound/machines/triple_beep.ogg', ASSEMBLY_BEEP_VOLUME, TRUE)
+	if(!src.quiet)
```

Comment:
```suggestion
	if(!quiet)
```
unnecessary
src.wires a couple lines above is also unnecessary but old code has settled

Reply:
Honestly why not just overrriding this proc (receive_signal) in the child? instead of doing this check.